cmake_format/configuration.py

```diff
@@ -242,6 +255,14 @@ def linewidth(self):
     "literal_comment_pattern":
     "If comment markup is enabled, don't reflow any comment block which matches"
     "this (regex) pattern. Default is `None` (disabled).",
+    "keep_comment_line_blocks":
```

Comment:
Seems this flag can be inferred from `line_block_min_length`. Also, I think "hash ruler" is a more meaningful name for the concept we're dealing with here. What do you think?

Reply:
Absolutely!  It's also much more consistent, "ruler" is already well defined.